common/rakelib/utils.rake

```diff
@@ -0,0 +1,74 @@
+# Used constants:
+# none
+
+class Utils
+  # formatter types
+  :xcpretty   # pass through xcpretty and store in artifacts
+  :raw        # store in artifacts
+  :to_string  # run using backticks and return output
+
+  # run a command using xcrun and xcpretty if applicable
+  def self.run(cmd, task, subtask = '', xcrun: false, formatter: :raw)
+    commands = xcrun ? [*cmd].map { |cmd|
+      "#{version_select} xcrun #{cmd}"
+    } : [*cmd]
+
+    case formatter
+    when :xcpretty then xcpretty(commands, task, subtask)
+    when :raw then plain(commands, task, subtask)
+    when :to_string then `#{commands.join(' && ')}`
+    else raise "Unknown formatter '#{formatter}'"
+    end
+  end
+
+  # print an info header
+  def self.print_info(str)
```

Comment:
Seing that other `log_info` function below make me realise that the name `print_info` for this function isn't the best either. It's more like `print_header` for example.

And an actual `print_info` function could be nice — typically to print the message `Finished installing. Binary is available in: …` at the end of the `cli:install` task — to print a message but without the `==` headers and possibly in a different color than those headers. Like green (32)  if OK, red (31) if KO (I imagine a `print_info(msg, error: false)` signature for example)